sklearnex/tests/test_n_jobs_support.py

```diff
@@ -32,7 +32,7 @@
     )
 )
 
-X, Y = make_classification(n_samples=40, n_features=4, random_state=42)
+X, y = make_classification(n_samples=40, n_features=4, random_state=42)
```

Comment:
`Y` was capitalized because it was constant and in main scope of script.
Code style changes are not subject to this PR, so it should remain without code change.